conf/modules.config

```diff
@@ -264,43 +264,43 @@ process {
     }
```

Comment:
The CAT_FASTQ_UNMAPPED|CAT_FASTQ_MAPPED that this closing bracket is assoicated with should be moved to after mapping and around the bam filtering, I see why you got confused by this now (I guess it was a straight copy paste from me rather than considering the position in modules.config, sorry!)

Reply:
Yes it is,